Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -195,7 +198,7 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section
```

Comment:
As the application stands right now, that wouldn’t be feasible, because the date picker is removed for the search view, so the view changes. The search view would need to somehow be integrated into the main viewController.